src/NServiceBus.Persistence.AzureTable/SagaPersisters/SecondaryIndices/SecondaryIndex.cs

```diff
@@ -31,26 +32,25 @@ public SecondaryIndex(bool assumeSecondaryIndicesExist = false, bool assumeSecon
 
             var rowKey = assumeSecondaryKeyUsesANonEmptyRowKeySetToThePartitionKey ? key.PartitionKey : key.RowKey;
 
-            TableResult exec;
+            Response<SecondaryIndexTableEntity> exec;
```

Comment:
Is there a behavioral difference between directly using the `T` of response or response?